team-validator.js

```diff
@@ -831,6 +834,7 @@ class Validator {
 						// chainbreeding with itself
 						// e.g. ExtremeSpeed Dragonite
 						if (!atLeastOne) {
+							if (noPastGen) continue;
```

Comment:
I believe they are. I just realized there is still a couple of checks missing when it comes to breeding with evolutions/preevolutions, working on it now.